src/main/java/uk/gov/hmcts/opal/controllers/LoggingExceptionResolver.java

```diff
@@ -1,33 +0,0 @@
-package uk.gov.hmcts.opal.controllers;
-
-import jakarta.servlet.http.HttpServletRequest;
-import jakarta.servlet.http.HttpServletResponse;
-import lombok.RequiredArgsConstructor;
-import lombok.extern.slf4j.Slf4j;
-import org.springframework.security.access.AccessDeniedException;
-import org.springframework.stereotype.Component;
-import org.springframework.web.servlet.HandlerExceptionResolver;
-import org.springframework.web.servlet.ModelAndView;
-import uk.gov.hmcts.opal.authentication.service.AccessTokenService;
-
-import static uk.gov.hmcts.opal.authentication.service.AccessTokenService.AUTH_HEADER;
-import static uk.gov.hmcts.opal.util.HttpUtil.extractPreferredUsername;
-
-@Component
-@RequiredArgsConstructor
-@Slf4j(topic = "LoggingExceptionResolver")
-public class LoggingExceptionResolver implements HandlerExceptionResolver {
```

Comment:
I think we might still need this class to log the AccessDeniedException message from addNote/SaveNote not being permitted

Reply:
AccessDeniedException is being handeled in GlobalExceptionHandler now as we were doing other exceptions